docs/src/examples/components/Alert/Usage/AlertExampleWidth.shorthand.tsx

```diff
@@ -0,0 +1,20 @@
+import { useRangeKnob } from '@stardust-ui/docs-components'
+import { Alert } from '@stardust-ui/react'
+import * as React from 'react'
+
+const AlertExampleWidth = () => {
+  const [width] = useRangeKnob({ name: 'width', initialValue: '500px' })
```

Comment:
Wouldn't it be better to specify the bounds on knob instead? Let's create a ticket for it?

Reply:
I am working on PR that will introduce it

Reply:
Here it is #1876.